src/main/java/docSharing/requests/CreateDocumentReq.java

```diff
@@ -14,4 +14,13 @@ public class CreateDocumentReq {
     private String name;
     private Long parentFolderId;
     private String content;
+
+    @Override
+    public String toString() {
+        return "CreateDocumentReq{" +
+                "name='" + name + '\'' +
+                ", parentFolderId=" + parentFolderId +
+                ", content='" + content + '\'' +
+                '}';
+    }
```

Comment:
We have @ToString annotation that spares us from using the toString function. Replace it.